priiloader/hacks_hash.ini

```diff
@@ -548,3 +548,15 @@ minversion=1
 amount=1
 hash=0x2c030000,0x418200ac,0x38000001,0x981f0010
 patch=0x480000b0
+[No-Block RAAE,408x,410x]
+maxversion=513
+minversion=417
+amount=1
+hash=0x28009ea3,0x4082009c
+patch=0x28009ea3,0x4082009c,0x4280006c
```

Comment:
where does this branch go to?

Reply:
It goes to the code that gets branched to assuming none of the blocked IDs match.

Reply:
what do those 4 function calls that it skips do? in a quick skim i didn't see the title id's ?

Reply:
They’re all strncmp calls that compare with strings stored in the r13 TOC. The first one is RAAE, the second is 408x, the third is 410x, and the fourth is 410x again.

Reply:
ah, the function is strncmp. fair enough